# Clustering_Analysis_Cancer. Date created/updated: December, 9, 2024.
[R: GEOquery, Biobase, preprocessCore, multiClust, ctc, gplots, dendextend, graphics, grDevices, amap].
[Paper Link:](https://link.springer.com/chapter/10.1007%2F978-3-030-45385-5_52)
[Poster Link:](https://www.claflin-computation.com/lab-journey?pgid=ktmii98q-ad0f6a9d-26d0-4daf-8ec1-ca96f20fba3c)
<img width="209" alt="Screenshot 2023-05-17 at 10 27 42 AM" src="https://github.com/spawar2/Clustering_Analysis_Cancer/assets/25118302/c212cdd7-4fb1-4f41-80a9-2c9d7cd31a01">
<img width="844" alt="IWBBIO" src="https://github.com/spawar2/Clustering_Analysis_Cancer/assets/25118302/a8420e56-43b9-463a-943b-b6585662a13d">
<img width="159" alt="Clustering" src="https://github.com/spawar2/Clustering_Analysis_Cancer/assets/25118302/8ce6753f-7fe8-4e3d-b4d4-4111a071649b">
Clustering Reveals Common Check-Point and Growth Factor Receptor Genes Expressed in Six Different Cancer Types, Shrikant Pawar and Stanam, A., Publication: Springer: Lecture Notes in Computer Science (Publication date: April 30), collaboration with Dr. Lahiri, Sunway University, Malaysia, Github, Article link, [Cited times: 2]^^^ [**2020:6].1,2,3,4. DOI: https://doi.org/10.1016/S2665-9913(23)00098-X, Issue: 5, Volume: 6, Pages: e316-e329.
[8th International Work-Conference on Bioinformatics and Biomedical Engineering, Granada, Spain, 2 February, 2020. Clustering reveals common check-point and growth factor receptor genes expressed in six different cancer types, by Pawar SD, Stanam A and Lahiri C.](https://iwbbio.ugr.es/).
Springer Bioinformatics and Biomedical Engineering.
https://campuspress.yale.edu/shrikantpawar/files/2024/05/8-IWBBIO.pptx
Video presentation: https://www.youtube.com/watch?v=Y6skvhHVR2w&ab_channel=ShrikantPawar
Claflin University, Orangeburg, South Carolina, United States of America (USA). 
https://www.claflin.edu/ Digital Object Identifier (DOI)=Online International Standard Book Number (ISBN).
https://www.claflin.edu/academics-research/schools-departments/school-of-natural-sciences-and-mathematics/department-of-mathematics-computer-science/computer-science
†
†Corresponding author. ††First author. †††Second author. ††††Third author. †††††author.
Cancer_Clustering.R: Breast, Colon, Lung, Oesophageal, Multiple Myeloma, Ovarian Microarray data read, robust multi array (RMA) Normalization, Kmeans analysis, Hierarchal clustering, Plotting.
selected function(merge, cluster_analysis, hclust, cutree, rbind, heatmap.2, setwd, read.csv, library, set.seed, sample.split, subset, na.omit, scale, svm, predict, table, plot).
Associated Press Style

https://www.java.com/
https://www.php.net/
https://www.mysql.com/
https://www.mathworks.com/products/matlab.html

cytoHubba: https://apps.cytoscape.org/apps/cytohubba
https://www.r-project.org/
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html
https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html
https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html
https://ggplot2.tidyverse.org/
https://cran.r-project.org/package=randomForest
https://topepo.github.io/caret/
https://bioconductor.org/packages/release/bioc/html/DESeq2.html
https://cran.r-project.org/package=missForest
https://bioconductor.org/packages/release/bioc/html/affy.html
https://bioconductor.org/packages/release/bioc/html/limma.html
https://rstudio.github.io/DT/
https://bioconductor.org/packages/release/bioc/html/edgeR.html
https://apps.cytoscape.org/apps/cytohubba
https://cytoscape.org/
https://string-db.org/
https://www.gsea-msigdb.org/
https://bioconductor.org/packages//2.13/bioc/html/BGmix.html
https://ggplot2.tidyverse.org/
https://www.htmlwidgets.org/
https://cran.r-project.org/package=cluster
https://cran.r-project.org/package=fpc
https://cran.r-project.org/package=survival
https://www.bioconductor.org/packages/release/bioc/html/multiClust.html
https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html
https://cran.r-project.org/package=dendextend
https://cran.r-project.org/package=amap
https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://shiny.posit.co/
https://cran.r-project.org/package=ggbeeswarm
https://cran.r-project.org/package=speedglm
https://magrittr.tidyverse.org/

https://www.python.org/
https://scikit-learn.org/
https://pandas.pydata.org/
https://matplotlib.org/
https://www.tidyverse.org/
https://numpy.org/
https://www.tensorflow.org/
https://kivy.org/
https://pydicom.github.io/
https://docs.python.org/3/library/shlex.html
https://scipy.org/
https://scikit-image.org/
https://seaborn.pydata.org/
https://librosa.org/doc/
https://docs.python.org/3/library/glob.html
https://pytorch.org/
https://pydicom.github.io/
https://biopython.org/docs/dev/api/Bio.Blast.html
https://pypi.org/project/openai/0.26.5/
https://docs.python.org/3/library/tkinter.html
https://docs.fast.ai/
https://pypi.org/project/torchvision/
